roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -115,6 +119,19 @@ def execute_module(self, module_name=None, module_args=None):
             )
         return self._execute_module(module_name, module_args, self.tmp, self.task_vars)
 
+    def execute_module_with_retries(self, module_name, module_args):
+        """Run execute_module and retry on failure."""
+        result = {}
+        tries = 0
+        while True:
+            res = self.execute_module(module_name, module_args)
+            if tries > self._module_retries or not res.get("failed"):
+                result.update(res)
+                return result
+            result["last_failed"] = res  # NOTE don't set result as its own member
```

Comment:
What's this note about?

Reply:
[This comment](https://github.com/openshift/openshift-ansible/pull/5125#discussion_r133969151) (note, I borrowed a commit from that PR since it's stalled on completely different stuff)

Really wasn't sure how to convey this without a lot of explanation. If it's more confusing than helpful I could just omit the comment...

Reply:
Not obvious to me where the recursion problem come from? The current comment != the linked comment, and even knowing both I still don't get the subtlety involved :)

Reply:
Self-referential structures. Honestly it didn't seem like much of an issue to me either. I think this is more confusing than helpful, will just nuke the comment.